### PR TITLE
Update labeller routes.

### PR DESCRIPTION
**Pull request by analogrelay**
Updated some labeller routing logic.

Not sure how the WebSocket one will interact with the Middleware one, but I guess we'll see.

### CODE REVIEW

**Comment by BrennanConroy** [comment]
> Not sure how the WebSocket one will interact with the Middleware one, but I guess we'll see.

I assume it'll add both

File: .github/labeler.yml [hunk 1]
```diff
@@ -39,15 +44,20 @@ area-mvc:
...
+area-platform:
+  - src/Framework/**/*
 
 area-security:
   - src/Security/**/*
+  - src/Azure/AzureAD/**/*
 
+# For now, "Http" is under servers. We might adjust that later.
 area-servers:
   - src/Servers/**/*
```
Comment (BrennanConroy) [inline .github/labeler.yml]: We could try applying the servers-* label for specific sub areas?

**Review by BrennanConroy — APPROVED** [review]

**Review by Tratcher — APPROVED** [review]

File: .github/labeler.yml (hunk 1 above)
Comment (anurse) [reply .github/labeler.yml]: I don't really use those anymore. I don't think they're that valuable.

**Comment by anurse** [comment]
> I assume it'll add both

Yep, that's my assumption too. It's OK if it does that I guess.

**Pull request merged**